### ISSUE TITLE
Finetuning parameters

### ISSUE THREAD

**Issue by okacopter** [Quint-e/equivariant-self-supervision-tempo#5]
Thank you for your excellent research !
I performed fine-tuning using your pre-training model(TCN-pretrained-220204-1729-002-8b0f9fa5.pt and TCN-pretrained-220204-1729-002-8b0f9fa5_config.yaml) to test Hainsworth, Giant Steps, and ACM_Mirum. However, I was unable to achieve Acc1 and Acc2 equivalent to those in Table 1 of the paper. Note that I achieved equivalent accuracy for GTZAN. Are there any special methods or hyperparameters required for fine-tuning Hainsworth, Giant Steps, and ACM_Mirum?

**Comment by okacopter** [comment]
For example, when testing Hainsworth, we used gtzan, giantsteps, and ACM_mirum for fine-tuning. The hyperparameters were not changed from finetune.yaml.
